frontend/src/containers/Routing/index.tsx

```diff
@@ -31,6 +33,10 @@ const Routing: React.FunctionComponent<IRoutingProps> = ({ isLoading }) => (
       <PublicRoute exact path="/" component={LandingPage} />
       <PublicRoute exact path="/login" component={LoginPage} />
       <PublicRoute exact path="/main" component={MainStudentPage} />
+      <PublicRoute exact path="/public" component={Data} />
+      <PublicRoute exact path="/add_path" component={AddPathPage} />
+      <PublicRoute exact path="/landing" component={LandingPage} />
+      <PublicRoute exact path="/add_course" component={AddCourse} />
       <PublicRoute exact path="/oauth2/redirect" component={handler} />
```

Comment:
Rename it to `<PublicRoute exact path="/oauth2/redirect" component={handler} />` or update with the latest dev.